handbook/product/surfacing_product_feedback.md

```diff
@@ -2,28 +2,26 @@
 
 > If you're working with a specific customer on a bug or issue and need to directly involve a product engineering team, please refer to [engaging other teams](../support/engaging-other-teams.md).
 
-CEs, CSEs, sales, marketing, engineering, and other teammates that interact directly with Sourcegraph users and/or Sourcegraph itself should share product feedback with the product team.
+CEs, CSEs, Sales, Marketing, Engineering, and other teammates that interact directly with Sourcegraph users and/or Sourcegraph itself should share product feedback with the product team.
 
 We deeply value this feedback, so we make this process as frictionless for teammates as possible:
 
-## Post the feedback in [`#feedback`](https://sourcegraph.slack.com/archives/C0W2E592M) on Slack
+## Create a Product Gap and a Submission in Salesforce
 
-You're welcome to tag teammates or teams, but that's not necessary – once the feedback is in [#feedback](https://sourcegraph.slack.com/archives/C0W2E592M) the product team will [make sure it's routed correctly](product_management/responding_to_user_feedback.md#slack-feedback-channel).
+If you have gotten a specific feedback requests that a customer(s) would like to track the progress of you should log that feedback as a Product Gap in Salesforce. Doing so allows full transparency to CEs and Product Eng on how many customers are impacted by an issue and what teams are responsible.
 
-## When to create a GitHub issue
+To create a Product Gap, navigate to the impacted customer(s) Account and whatever current open Opportunity they have, whether it's a net new opp or a renewal. When vieing the Opportunity, you should see a related list labeled "Product Gap Submissions". Create a new Submission, and search in the field Product Gap, to see if a Gap already exists for what you wish to log. If no gap exists, create one, and then continue creating your submission that will tie your Customer to the Product Gap. Be sure to include relevant information like what workarounds your customer must employ (if any), and the severity of what this issue blocks for them.
```

Comment:
Not a big deal but should it be "most recent open opportunity they have"?

vieing => viewing

"and search in the field Product Gap, to see if a Gap already exists for what you wish to log" 
I don't think you need a comma after Product Gap

inconsistent capitalization of gap on its own, probably capitalize Product Gap, but not gap